README.md

```diff
@@ -1,4 +1,4 @@
-# strapPy
+# StrapPy
```

Comment:
```suggestion
# Strappy
```

Reply:
@BooleanJulien that's intentional, just followed the NumPy convention for readme after checking their repo